CurrencyConverterApplicationTests/UnitTests/CurrencyConverterTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using CurrencyConverterControl;
+using CurrencyConverterControl.Model;
+using NUnit.Framework;
+using System;
+using System.Windows.Controls;
+
+
+namespace CurrencyConverterApplicationTests.UnitTests
+{
+    [TestFixture]
+    public class CurrencyConverterTests
+    {
+      
+
+        [Test]
+        [Apartment(ApartmentState.STA)]
+        public void ConvertValues_SuccessfulRequest_ShouldReturnPositiveConvertedAmount() 
+        {
+            // Arrange
+            var converterControl = new CurrencyConverterControl.CurrencyConverterControl();
+
+            // Act
+            converterControl.InputValue = 100;
+            converterControl.SourceCurrency = "USD";
```

Comment:
What if I SourceCurrency differs from the USD?

Reply:
this is to check the conversion method used for the data grid cells. In fact, as in the requirements it was that the Prices of the Data Grid elements will be in USD, and there was an API which had all the exchange rates for USD as a Source, l have used that endpoint to execute the conversions.

Reply:
in addition, if it differs l have to test the Convert method. I have tested it with the 
ConvertValues_FailedRequest_ShouldReturnValueGreaterThanConvertedAmount Test.

Reply:
>as in the requirements it was that the Prices of the Data Grid elements will be in USD

This is the requirements for the sample. But we're talking about control. You shouldn't create a control for only one case. 
This is wrong. The main goal is to create a good control, not a sample. The sample is just a demo of your control. 

You have public API that allows to use different currencies. You have to support these cases and check that they work. Not necessarily to test every currency, but some kind of selection.